DependencyInjection/MonologExtension.php

```diff
@@ -652,6 +652,13 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 $handler['app_name'],
             ));
             break;
+        case 'server_log':
+            $definition->setArguments(array(
+                $handler['host'],
```

Comment:
In the `ServerLogHandler` class, the argument is named `$connectionString`. I would unify that.